Source/ACE/Entity/Landblock.cs

```diff
@@ -729,6 +733,13 @@ public void ScheduleItemTransfer(ActionChain chain, ObjectGuid wo, ObjectGuid co
             }
         }
 
+        public bool ItemExistsInWorld(ObjectGuid wo)
```

Comment:
I don't like this function.  Its dangerous to access the landblock, as items can be added to or removed from it arbitrarily.  If its only called in certain circumstances it may be safe, but in general its dangerous.